src/airport_gui.cpp

```diff
@@ -71,6 +71,7 @@ struct BuildAirToolbarWindow : Window {
 	BuildAirToolbarWindow(WindowDesc *desc, WindowNumber window_number) : Window(desc)
 	{
 		this->InitNested(window_number);
+		this->OnInvalidateData();
```

Comment:
why is this necessary?

Reply:
1) consistency with all the other Toolbar GUIs
2) without it, `OnInvalidateData` is not called when opening the toolbar, not updating properly

Not sure if there are better ways, I just stole it from the other Toolbar GUIs honestly :)